airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/initialsync/MySqlInitialLoadRecordIterator.java

```diff
@@ -213,6 +213,11 @@ public void close() throws Exception {
     }
   }
 
+  @Override
+  public Optional<AirbyteStreamNameNamespacePair> getAirbyteStream() {
```

Comment:
Is a similar change required on other connectors?

Reply:
Yes. That will have been reflected in the PRs associated with WASS changes in the relevant connectors